pyh5md/core.py

```diff
@@ -167,7 +171,7 @@ def observable(self, obs_name, shape=None, dtype=None, data=None, time=True, uni
                 return TimeData(self.f['observables'],obs_name, shape=shape, dtype=dtype, data=data, unit=unit, time_unit=time_unit, **kwargs)
             else:
                 return FixedData(self.f['observables'],obs_name, shape=shape, dtype=dtype, data=data, unit=unit, **kwargs)
-        
+
```

Comment:
Hi Jakub,

Even though the trailing space is wrong I prefer to leave minor style issues separated from actual changes to the code. Can you edit your PR so that only lines 137-140 are modified?